app-config.json

```diff
@@ -682,11 +682,19 @@
                   },
                   "dpp": {
                     "validUntil": "2025-11-28T04:47:15.136Z",
-                    "context": ["https://test.uncefact.org/vocabulary/untp/dpp/0.5.0/"],
+                    "context": [
+                      "https://test.uncefact.org/vocabulary/untp/dpp/0.5.0/",
+                      {
+                        "ex": "https://www.w3.org/2018/credentials#renderMethod#",
+                        "renderMethod": "https://www.w3.org/2018/credentials#renderMethod",
+                        "template": "ex:template",
```

Comment:
This term, `template`, may well need defining, but I don't think here is the place to do it. And defining it as `"ex:template"` here is effectively giving it the `id` of `https://www.w3.org/2018/credentials#renderMethod#template`, which is also not correct, I don't think? (it's not defined as part of the VCDM renderMethod defs, as far as I can see?)

Anyway, I've created https://github.com/uncefact/spec-untp/issues/309 as, if we want to use a `template` in the credentials, it needs to be included in our own context somehow.

Reply:
Great, thanks for the clear explanation. So yes, we need to update the models to ensure that the expanded URI for `template` and `url` here is correct (it won't be currently in 0.6.0-beta5, and it doesn't exist in 0.5.0). I'll try to update that, but for now, yes, best to include `template` and `url` here explicitly. Please still do try adding https://www.w3.org/ns/credentials/v2 to the context rather than defining `renderMethod` explicitly, though.